warehouse/warehouse.go

```diff
@@ -1811,6 +1813,24 @@ func Start(ctx context.Context, app app.Interface) error {
 		}))
 	}
 
+	// Report warehouse features
+	g.Go(func() error {
+		backendconfig.DefaultBackendConfig.WaitForConfig(ctx)
+
+		c := features.New(
```

Comment:
it took me a while to understand that we are creating a new features client here, thus I propose to rename `features.New` to `features.NewClient`